src/subscript/sector2fluxnum/flux_util.py

```diff
@@ -66,96 +65,6 @@ def filter_region(grid, i, j, k, fipnum, fipnum_kw, combine_operator="intersect"
         )
 
 
-def filter_region_fipnum(grid, fipnum, fipnum_kw, combine_operator="intersect"):
```

Comment:
This function is never needed in the script?

Reply:
Probably. The `flux_util.py` is a collection of utility functions, and some of them are superfluous.